### DISCUSSION TITLE
Double hantek

### DISCUSSION THREAD

**Discussion by VictorGerin** [OpenHantek/OpenHantek6022#167] [Ideas]
First all, let me say how amazing is this project e even "unstable" is better than the original hantek software. Thanks All for this :)

as the simplest hantek oscilloscope (6022BE) has only two input could be possible use 2 6022BE as was only one oscilloscope with 4 input ports ?

**Comment by Ho-Ro** [comment]
My 1st thought ist: Not possible, because you have two systems with own clocks which are not in sync. But you can work with two scopes at the same time.

**Reply by VictorGerin** [reply]
That was i thought too probably a hardhack only e maybe only :/

any way thanks :)

**Comment by Ho-Ro** [comment]
Another hindering point could be the use of four channels, I tried to keep the channel count variable, but never checked more (or even less) than 2 channels.
